src/plone/restapi/tests/http-examples/controlpanels_get_dexterity_item.resp

```diff
@@ -111,7 +111,6 @@ Content-Type: application/json
       }, 
       "description": {
         "description": "", 
-        "minLength": 0, 
```

Comment:
@avoinea ok, it makes sense to remove that if the value is 0. Technically this is a breaking change but I guess nobody uses that in the first place.